test/test_parser.cpp

```diff
@@ -752,6 +752,7 @@ TEST(Parser_substitution)
     CHECK_THROW_ANY(verify_query_sub(test_context, t, "binary == $7", args, num_args, 0));
 }
 
+/*
```

Comment:
Currently fails in Jenkins; should be enabled again before merge. The unit test is not related to this PR.